### DISCUSSION TITLE
Best method to start tiddlywiki automatically on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by ubuntupunk** [TiddlyWiki/TiddlyWiki5#5779] [Q&A]
If I start tiddlywiki as a systemd service, it has issues with saving. I have a bash script in /usr/local/bin

#!/bin/bash
tiddlywiki ~/mywiki --listen

Instead of saving, the wiki downloads itself to downloads. I can't install themes it seems.

What is the best method to call tiddlywiki automatically upon login on Ubuntu?

**Comment by ualich** [comment]
Check your ~/mywiki folder. There should be a file tiddlywiki.info. Make sure plugins tiddlywiki/filesystem and tiddlywiki/tiddlyweb are listed in this file.

**Reply by ubuntupunk** [reply]
They all there "plugins": [
        "tiddlywiki/tiddlyweb",
        "tiddlywiki/filesystem",

**Reply by ubuntupunk** [reply]
I reinstalled tiddlywiki via npm and also gave it a direct link /home/user/mywiki 
works fine now thanks

**Comment by linonetwo** [comment]
You can try https://github.com/tiddly-gittly/TiddlyGit-Desktop , it have an option to start on startup. And it runs nodejs wiki.
